public class ExceptionExample {
    public static void main(String[] args) {
        int numerator = 10;
        int denominator = 0;

        System.out.println(numerator);
        System.out.println(denominator);

        try {
            int result = numerator / denominator;
            System.out.println("result: " + result);
        } 
        catch (ArithmeticException e) { 
            System.out.println("Error occurred");
        }
    }
}
